plugins/styleguide/plugin.rb

```diff
@@ -13,3 +13,9 @@
 Discourse::Application.routes.append do
   mount ::Styleguide::Engine, at: '/styleguide'
 end
+
+after_initialize do
+  register_asset_filter do |type, request|
+    request.fullpath && request.fullpath.start_with?('/styleguide')
```

Comment:
```suggestion
    request.fullpath&.start_with?('/styleguide')
```

"SNOT": safe navigation operator, try (it) 😝

Reply:
Should be double & actually, because request can be nil

Reply:
If the request can be nil, then `request.fullpath` might fail, so should be `request&.fullpath&.start_with...`